### PR TITLE
Issue #442: Language page layout

### PR DESCRIPTION
**Pull request by phillip-hopper**
As a user I want to have a home page for my language so that I can just see content in my language and so that I can understand the page.

See mockups on #27.

### CODE REVIEW

File: _layouts/i18n.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+<script type="html/template" id="more-template">
+  <div class="more-container">
+    <span><a href="" class="bold">{{ site.data[page.language_code].l10n.view_more }}</a></span>
+  </div>
+</script>
+
+<header id="pinned-header" class="pin-to-top">
+  <div class="container">
+    <div class="float-left">
```
Comment (richmahn) [inline _layouts/i18n.html]: While this is float left, the Popular/Recent tab section should be completely to the left with a little left margin. Probably needs a 100% width of its parent container in the header? Looks best if "Popular" is left-justified to be right about the header of the "Popular" section of the page.

File: _layouts/i18n.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      <table>
+        <tr>
+          <td><span class="bold">{{ site.data[page.language_code].l10n.popular }}</span>&emsp;<span class="bold">{{ site.data[page.language_code].l10n.recent }}</span></td>
```
Comment (richmahn) [inline _layouts/i18n.html]: The Popular and Recent links at the top should link to the respective headings on the page.

File: _layouts/i18n.html (hunk 1 above)
Comment (richmahn) [inline _layouts/i18n.html]: See mockup: https://cloud.githubusercontent.com/assets/21183377/20073735/0a383d1e-a4f3-11e6-9ec3-42975e2a219c.png

Comment (phillip-hopper) [inline _layouts/i18n.html]: @richmahn The mockup is a narrow screen. and the text in the header is also lined up with the text below the header, which is the metric I used.

File: _layouts/i18n.html (hunk 2 above)
Comment (phillip-hopper) [inline _layouts/i18n.html]: In the mockup all other clickable items are light blue, not black.

File: _layouts/i18n.html (hunk 1 above)
Comment (phillip-hopper) [inline _layouts/i18n.html]: Oh, I seem to have broken this. Will fix.

File: _layouts/i18n.html (hunk 2 above)
Comment (phillip-hopper) [inline _layouts/i18n.html]: Will make this text blue and clickable.

Comment (richmahn) [inline _layouts/i18n.html]: Yeah, blue would make it more clear that it is a tab/link.

**Pull request merged**